src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -132,6 +138,63 @@ private static bool HeadersAllowGzip(IDictionary<string, StringValues> headers,
             return false;
         }
 
+        protected async Task WaitForExecutionOnSpecificNode(TransactionOperationContext context, ClusterTopology clusterTopology, string node, long index)
+        {
+            using (var requester = ClusterRequestExecutor.CreateForSingleNode(clusterTopology.GetUrlFromTag(node), ServerStore.Server.Certificate.Certificate))
```

Comment:
Why is this an instance method that also create a new _instance_ for this?
What does this give us?